### PR TITLE
feat(*): formulae + cicero 0.21 + md transform 0.12 - I69

### PR DESCRIPTION
**Pull request by jeromesimeon**
### Changes
- Includes all changes from the `formulae` branch + some fixes for the newly released Cicero 0.21
- Additional changes are in this commit: https://github.com/accordproject/web-components/pull/161/commits/3fecfd0ed92c1ae0ae425e55c84cd464299d0eaa
- Comments ad-hoc code in story for contract editor, assuming a specific template with formulae
- Updates versions in story for template library
- Reverts default template in contract editor story to be late delivery and penalty
- Reverts temporary link to pre-release templates to use the default template library

### Flags
- Not a lot of testing
- Unclear status of the rest of the code (haven't reviewed all the prior changes)
- Some warnings from console below:

Screenshot 2020-08-03 at 2 57 45 PM

### CODE REVIEW

File: packages/storybook/src/stories/4-ContractEditor.stories.js [hunk 1]
```diff
@@ -71,27 +112,77 @@ This is text. This is *italic* text. This is **bold** text. This is a [link](htt
...
+
+    const found = val.children[1].children.filter(element => element.type === 'formula' && element.data.name === 'formula');
+    
+    action('Clause -> Parse: ')({
+      'Clause': ciceroClause,
+      'AST': ast,
+      'Draft': something
+    });
+
+    /* XXX What do we do with this? - JS
```
Comment (irmerk) [inline packages/storybook/src/stories/4-ContractEditor.stories.js:165]: Unsure about this. It's not commented out in [my branch](https://github.com/accordproject/web-components/blob/irmerk/i69/formulae-ui/packages/storybook/src/stories/4-ContractEditor.stories.js#L174)

Comment (jeromesimeon) [reply packages/storybook/src/stories/4-ContractEditor.stories.js:165]: I know! But it seems to assume a specific template, no? Do we really want to leave this code in for the release?

Comment (irmerk) [reply packages/storybook/src/stories/4-ContractEditor.stories.js:165]: I don't think so - just will need to come back to it.

**Comment by irmerk** [comment]
Related for context:
https://github.com/accordproject/web-components/pull/155
https://github.com/accordproject/web-components/pull/123

**Pull request merged**